### DISCUSSION TITLE
Repository access

### DISCUSSION THREAD

**Discussion by ertrzyiks** [hexojs/hexo#5508] [General]
Hey @hexojs/core , I lost my admin rights to the 3 repos I transferred to the hexo organization:

- https://github.com/hexojs/hexo-filter-responsive-images
- https://github.com/hexojs/hexo-filter-lqip
- https://github.com/hexojs/hexo-fontawesome

Those are not actively developed in a sense of new features being added, but I keep them in the good shape to my best effort. 

I just published 1.9.0 of https://www.npmjs.com/package/hexo-filter-responsive-images but I can't push commits to the master branch of https://github.com/hexojs/hexo-filter-responsive-images

My access was previously granted by a dedicated "Mateusz Derks" team in the Hexo org but I received an email that the team was removed. I guess my access was not reintroduced in any other way.

**Comment by uiolee** [comment]
You have higher access now.

**Comment by ertrzyiks** [comment]
Thanks @uiolee , I got back enough access to proceed supporting the repos 👍
